services/managers/fleetup_manager.py

```diff
@@ -0,0 +1,141 @@
+from django.conf import settings
+from datetime import datetime
+
+import logging
+import requests
+import json
+
+appkey = settings.FLEETUP_APP_KEY
+userid = settings.FLEETUP_USER_ID
+apiid = settings.FLEETUP_API_ID
+groupid = settings.FLEETUP_GROUP_ID
+
+class FleetUpManager():
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def get_fleetup_members():
+        url = "http://api.fleet-up.com/Api.svc/" + str(appkey) + "/" + str(userid) + "/" + str(apiid) + "/GroupCharacters/" + str(groupid) + ""
+        try:
+            jsondata = requests.get(url).content
+        except requests.exceptions.ConnectionError:
+            r.status_code = "Connection timeout"
+        fmembers=json.loads(jsondata.decode())
+
+        return {row["UserId"]:{"user_id":row["UserId"],
+                               "char_name":row["EveCharName"],
+                               "char_id":row["EveCharId"],
+                               "corporation":row["Corporation"]} for row in fmembers["Data"]}
+
+    @staticmethod
+    def get_fleetup_operations():
+        url = "http://api.fleet-up.com/Api.svc/" + str(appkey) + "/" + str(userid) + "/" + str(apiid) + "/Operations/" + str(groupid) + ""
+        try:
+            jsondata = requests.get(url).content
+        except requests.exceptions.ConnectionError:
+            r.status_code = "Connection timeout"
+        foperations=json.loads(jsondata.decode())
```

Comment:
this is undefined if an exception occurs. Maybe the `except` block should return?